playwright.config.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+const MAX_RETRIES = 3;
+const isCI = process.env.CI === "true";
+
+/** @type { import('@playwright/test').PlaywrightTestConfig} */
+module.exports = {
+  globalSetup: require.resolve("./scripts/setupPlaywright.js"),
+  testIgnore: "**/*.ignore.*",
+  testDir: "./test/e2e",
+  snapshotPathTemplate: "./test/e2e/__snapshots__/{testFilePath}/{arg}{ext}",
+  fullyParallel: false,
```

Comment:
Ideally you enable fullyParallel - didn't it work with having it enabled? Might be a potential follow-up.

Reply:
yes, it didn't work at first since we need to set ports, we planed to benefit from it later on.